styles/typography/typography-preset-5.json

```diff
@@ -0,0 +1,36 @@
+{
+	"version": 3,
+	"$schema": "https://schemas.wp.org/trunk/theme.json",
+	"title": "Literata & Ysabeau Office",
+	"slug": "typography-preset-5",
+	"styles": {
+		"typography": {
+			"fontFamily": "var:preset|font-family|ysabeau-office",
+			"fontSize": "var:preset|font-size|medium",
```

Comment:
I see the font size for this one as 24px in the designs.

Reply:
There is no preset for 24px.